src/cmd/utils.go

```diff
@@ -362,6 +362,8 @@ func getUsageForCommonCommands() string {
 	builder.WriteString("create    Create a new Toolbx container\n")
 	builder.WriteString("enter     Enter an existing Toolbx container\n")
 	builder.WriteString("list      List all existing Toolbx containers and images\n")
+	builder.WriteString("export    Export existing Binaries and Applications from Toolbx containers to the Host\n")
+	builder.WriteString("unexport  Unexport Toolbx containers exported Binaries and Applications from the Host\n")
```

Comment:
Nitpick: I'd leave it out from this list.

It's meant to be a starting point to orient the user with the most fundamental commands.  We don't list every command here (eg., `rm`, `rmi` and `run` are absent) because the full list can be obtained via tab completion or the manual pages.